ros/processor/inventory_events_consumer.py

```diff
@@ -87,6 +87,12 @@ def run(self):
         LOG.warning("Stopping inventory consumer")
         self.consumer.close()
 
+    def update_event_without_platform_metadata(self, msg):
+        if msg["platform_metadata"] is None and msg["type"] == "updated" \
+                and msg["host"]["system_profile"]["cloud_provider"] == "aws":
```

Comment:
I am not sure if I should gate on "aws" specifically here, we dont use to do it before!